packages/live-share-canvas/src/input/PointerInputProvider.ts

```diff
@@ -87,14 +85,12 @@ export class PointerInputProvider extends InputProvider {
 
                 const coalescedEvents = getCoalescedEvents(e);
 
-                coalescedEvents.forEach(
-                    (evt: PointerEvent) => {
-                        this.pointerMoveEvent.emit(
-                            {
-                                ...pointerEventToIPointerEvent(evt),
-                                isPointerDown
-                            });
+                coalescedEvents.forEach((evt: PointerEvent) => {
```

Comment:
Much less clear than before

Reply:
how so?

Reply:
@corinagum please see my response to James in a previous comment about the same thing.